src/shims/ApiCompat.proj

```diff
@@ -54,11 +54,32 @@
 
     <Error Condition="'$(ApiCompatExitCode)' != '0'" Text="ApiCompat failed comparing netfx to $(TargetGroup)" />
 
+
+    <!--
+      In order to update the .NET Standard baseline, you can just start the build with either
+
+        /p:UpdateNETStandardBaseline=True
+
+      of
+
+        /p:UpdateNETStandardBaselineLocally=True
+
+      The difference is that the former uses the official .NET Standard while the latter uses a locally built
+      .NET Standard (Debug, in a folder parallel to where you checked out CoreFx).
+    -->
     <PropertyGroup>
+      <UpdateNETStandardBaseline Condition="'$(UpdateNETStandardBaseline)' == ''">False</UpdateNETStandardBaseline>
+      <UpdateNETStandardBaselineLocally Condition="'$(UpdateNETStandardBaselineLocally)' == ''">False</UpdateNETStandardBaselineLocally>
       <NETStandard21OnlyRef>$(NetStandard21RefPath)netstandard.dll</NETStandard21OnlyRef>
+      <NETStandard21OnlyRef Condition="$(UpdateNETStandardBaselineLocally)">$(MSBuildThisFileDirectory)..\..\..\standard\artifacts\bin\ref\netstandard\Debug\netstandard.dll</NETStandard21OnlyRef>
```

Comment:
nit: consider `_camelCase` for properties you don't intend to be passed in / defined elsewhere.

Reply:
Good call, fixed with aaf1a0d687.